website/docs/r/alert_channel_splunk.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_splunk"
+description: |-
+  Create and manage Splunk Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_splunk
+
+You can use this resource enabled Lacework to forward alerts to Splunk using an HTTP Event Collector.
```

Comment:
this sentence sounds odd, is it missing a preposition? 🤔
```suggestion
You can use this resource to enable Lacework to forward alerts to Splunk using an HTTP Event Collector.
```

Reply:
Fixed